cmd/broker/ingress/main.go

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
How to add the `kodata` folder?

Reply:
What I learned - just copy it from another folder.